public/app/features/dashboard-scene/edit-pane/DashboardEditPane.tsx

```diff
@@ -108,6 +108,23 @@ export class DashboardEditPane extends SceneObjectBase<DashboardEditPaneState> {
     this.setState({ showAddPane: !this.state.showAddPane });
   }
 
+  public forceSelectObject(obj: SceneObject, id: string) {
```

Comment:
could you use selectObject method intead with a force?: boolean param when true would skip the toggling? or is there something else different with this implementation?

Reply:
I also thought about it but a `force` parameter kinda clashes with the `multi` parameter. We have some options though:
- make `selectObject` receive either `multi` or `force` parameters: `selectObject(obj: SceneObject, id: string, options?: { multi: boolean } | { force: boolean })`
- make `selectObject` receive both `multi` and `force` parameters and either:
   - ignore `multi` if `force` is `true`
   - make use of `multi` and just skip the deselect action if the element is already selected or add it to the selection

Not sure which one of these is the perfect option.

Reply:
Found a good way:
- pass `options` parameter instead of `multi`
- `multi` can be either manually controlled by user. If not specified, it defaults to `!!evt.shiftKey` (this way it can be programmatically disabled)
- if `force: true` and `multi: true`, it just makes sure that the element is selected